pandas/core/indexes/base.py

```diff
@@ -2022,18 +2022,48 @@ def hasnans(self):
 
     def isna(self):
         """
-        Detect missing values
+        Detect missing values.
+
+        Return a boolean same-sized object indicating if the values are NA.
+        NA values, such as None or :attr:`numpy.NaN`, get mapped to True
+        values.
+        Everything else get mapped to False values. Characters such as empty
+        strings `''` or :attr:`numpy.inf` are not considered NA values
+        (unless you set :attr:`pandas.options.mode.use_inf_as_na` `= True`).
 
         .. versionadded:: 0.20.0
 
         Returns
         -------
-        a boolean array of whether my values are NA
+        numpy.ndarray
+            A boolean array of whether my values are NA
 
         See also
         --------
-        isnull : alias of isna
+        pandas.Index.isnull : alias of isna
```

Comment:
could say this is a synonym

Reply:
Not my native language, I went with the language that was originally used. I can change it across all docs we have commited fi you'd like to, @jreback . Y/N?

Reply:
oh sorry, yes alias is fine.